### PR TITLE
[SW-2512] H2OMOJOPipelineMOJOModel Can Produce Predictions of Various Types

### PR DESCRIPTION
**Pull request by mn-mikke**
This PR is created because of DAI Big Data recipe and ability to use the output of DAI pipeline feature transforms which could be of type `string`.

### CODE REVIEW

File: doc/src/site/sphinx/migration_guide.rst [hunk 1]
```diff
@@ -49,6 +49,9 @@ From 3.32 to 3.32.1
 
 - The data type of H2OTargetEncoder models has been changed from ``DoubleType`` to
```
Comment (pkozelka) [inline doc/src/site/sphinx/migration_guide.rst:50]: this sentence looks unfinished

File: doc/src/site/sphinx/migration_guide.rst [hunk 2]
```diff
@@ -49,6 +49,9 @@ From 3.32 to 3.32.1
...
 
+- The sub-columns of the ``prediction`` column produced byt ``H2OMOJOPipelineModel`` could be of the type ``float`` instead  of ``double``
```
Comment (pkozelka) [inline doc/src/site/sphinx/migration_guide.rst:52]: `byt`: is it a typo?

**Review by pkozelka — COMMENTED** [review]
only minor notes

File: doc/src/site/sphinx/migration_guide.rst (hunk 1 above)
Comment (mn-mikke) [reply doc/src/site/sphinx/migration_guide.rst:50]: Thanks for catching it!

File: doc/src/site/sphinx/migration_guide.rst (hunk 2 above)
Comment (mn-mikke) [reply doc/src/site/sphinx/migration_guide.rst:52]: yes, thanks for catching it!

**Review by pkozelka — APPROVED** [review]

**Comment by mn-mikke** [comment]
@satai and @pkozelka Thanks a lot for your reviews!

**Pull request merged**